eng/pipelines/templates/steps/wrapped-use-python.yml

```diff
@@ -0,0 +1,26 @@
+parameters:
+  versionSpec: ''
+
+steps:
+  # use python 3.8 for tooling. packaging. platform.
+  - task: UsePythonVersion@0
+    displayName: "Use Python 3.8"
+    inputs:
+      versionSpec: 3.8
+
+  - pwsh: |
+      pip install packaging
+    displayName: Prep Environment
+
+  # select the appropriate version from manifest if present
+  - task: PythonScript@0
+    displayName: 'Check Version Spec Against Known Versions'
```

Comment:
This name needs to be adjusted to `Install Python Version`. I'll do so next time I'm at a machine with git.